### ISSUE TITLE
Proxy Issue

### ISSUE THREAD

**Issue by jtpereyda** [jtpereyda/ezoutlet#2]
On Windows, when I added the HTTP_PROXY environment variable, ezoutlet started using the proxy even for 192.168.*.* addresses, meaning it stopped working.

One potential solution is given here: http://www.decalage.info/en/python/urllib2noproxy
```
proxy_handler = urllib2.ProxyHandler({})
opener = urllib2.build_opener(proxy_handler)
urllib2.install_opener(opener)
return urllib2.urlopen(url, timeout=self._timeout).read()
```

Edit: Post originally mentioned HTTPS_PROXY, but HTTP_PROXY is the variable that matters.

**Comment by jtpereyda** [comment]
This is kind of a pain to test. Maybe there is a simpler library out there, like `requests`.

**Comment by jtpereyda** [comment]
This is kind of a pain to test. Maybe there is a simpler library out there, like `requests`.

**Comment by jtpereyda** [comment]
Fixed using requests lib in PR #13
